HACKING.md

```diff
@@ -433,6 +433,19 @@ maybe you need to replace system installed snap-seccomp with the one aligned to
 you are testing. To do this, simply backup `/usr/lib/snapd/snap-seccomp` and overwrite it with 
 the testing one. Don't forget to roll back to the original, after you finish testing.
 
+### Testing the snap userd agent
+
+To test the `snap userd --agent` command, you must first kill the current process if it is
```

Comment:
(nitipck) I would use "stop" instead of "kill" here and maybe provide an example here too? Like `sstemctl --user stop snapd.session-agent.service` or some `pkill` magic?

Reply:
The point is that the service itself is already stopped (if not, it would be running constantly, IIRC). Anyway, the daemon quits automagically after some inactivity time, so it's improbable that it's running.

Reply:
After thinking again about it, you are right. Done.